src/DynamoCore/packages.config

```diff
@@ -2,6 +2,7 @@
 <packages>
   <package id="Greg" version="1.0.6176.18754" targetFramework="net45" />
   <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net45" />
+  <package id="NUnit" version="3.8.1" targetFramework="net45" />
```

Comment:
why is this package added to Dynamo core?

Reply:
Oh, weird.